presto-jdbc/src/test/java/io/prestosql/jdbc/TestJdbcConnection.java

```diff
@@ -250,6 +263,17 @@ private Connection createConnection(String extra)
         return set.build();
     }
 
+    private static Map<String, String> listExtraCredentials(Connection connection)
+            throws SQLException
+    {
+        ResultSet rs = connection.createStatement().executeQuery("SELECT * FROM system.test.extraCredentials");
+        ImmutableMap.Builder<String, String> builder = ImmutableMap.builder();
+        while (rs.next()) {
+            builder.put(rs.getString(1), rs.getString(2));
```

Comment:
Change these to `getString("name")` etc. so it's clear to the reader what the fields are. The positions are unknown since the query is `SELECT *`.